### DISCUSSION TITLE
Stuck on the Getting Started guide - can't create the histogram

### DISCUSSION THREAD

**Discussion by emilyherrett** [opensafely/documentation#285] [Q&A] [answered]
Hi

Newbie here and I'm going through the Getting Started guide, working online on gitpod via web browser (using Chrome on an old Mac in case that is relevant).  Successfully got to the 'Add a chart' section but I must have done something wrong as the code won't run to generate the histogram.

I did steps 1 to 4 in the 'add a chart' section and everything looks correct (copied and pasted the commands so I think they're ok, plus the input.csv file looks ok).  But when I try and run the following command in the Terminal to run the project.yaml file:

$opensafely run run_all --force-run-dependencies 

it does nothing.  Clearly I have done something wrong but cannot figure out what it is!  No problems detected in the workspace.  Please let me know if you need more info/question unclear.  

Thanks in advance
Emily

**Comment by StevenMaude** [comment, marked as answer]
Hi @emilyherrett! :wave: Gitpod is still a really new idea that we're trying. We're keen to fix stumbling blocks that users are discovering :slightly_smiling_face:

Some questions to try and diagnose:

1. What does running just `opensafely` show?

2. Are you trying to run literally:

    `$opensafely run run_all --force-run-dependencies`

    or are you running:

    `opensafely run run_all --force-run-dependencies`

    following the terminal prompt `$`?

    You can see in this example the difference between the two:

    ![Running OpenSAFELY in Gitpod.](https://user-images.githubusercontent.com/3818079/122550032-1639f400-d02b-11eb-864c-2bfa0685f4bd.png)

3. What specifically happens when "it does nothing"? Is there an error message or nothing at all?

If you can post a screenshot of everything in Gitpod after you've attempted to run the `opensafely` command and it's failed, that might also help.

(I'm in office Tuesday-Thursdays. Happy to diagnose by a quick screensharing call then if there are still problems, but hope this or someone else's suggestions might get it running for you.)

**Reply by emilyherrett** [reply]
Hi Steven

Thanks for the helpful reply. 

Unhelpfully, I decided to take a break - so my mac went into in sleep mode.  When it woke up, the command ran first time and now I have my graph.  Whatever I had done before is sadly gone but is likely to be my mistake!

But to answer your questions above, I tried with and without the $.  And with and without spaces, and with and without the gitpod filepath - the cursor just moved onto the next line like I'd asked it to do nothing.  No error messages at all.   I'll keep going with the guide and will make sure to add screenshots next time.  

I'll be back pestering you if it happens again.

Thanks again for replying and for the offer of help 
Emily

**Reply by StevenMaude** [reply]
Great; glad you got it working and don't hesitate to ask if you have more questions! :slightly_smiling_face:
